test/jasmine/tests/hover_label_test.js

```diff
@@ -16,11 +16,8 @@ var fail = require('../assets/fail_test');
 describe('hover info', function() {
     'use strict';
 
-    var mock = require('@mocks/14.json'),
-        evt = {
-            clientX: mock.layout.width / 2,
-            clientY: mock.layout.height / 2
-        };
+    var mock = require('@mocks/14.json');
+    var evt = { xpx: 355, ypx: 150 };
```

Comment:
... making this test suite more robust. Previously, it relied on [this](https://github.com/plotly/plotly.js/blob/6e547ef794093c206998103f3065d5f2ab2a20b4/src/plots/cartesian/graph_interact.js#L422) fallback to get the hover pixel position.